tools/build/Library/InstallUnpacker/classes/VersionNumber.php

```diff
@@ -115,7 +115,7 @@ public function __toString()
      *
      * @return int 1 if this version number is higher, -1 if lower, 0 if equal
      */
-    public function compare(VersionNumber $otherNumber)
+    public function compare(self $otherNumber)
```

Comment:
not sure it is accepted in all versions of PHP, I've never seen this kind of use before: would you mind to revert it?

Reply:
I think it was introduced in PHP 5.3. But I can revert that part :-)

Reply:
Fixed.